src/locale/locales/es/messages.po

```diff
@@ -650,11 +650,11 @@ msgstr "Se produjo un error al generar su paquete de inicio. ¿Quieres intentarl
 
 #: src/view/com/util/post-embeds/VideoEmbed.tsx:135
 msgid "An error occurred while loading the video. Please try again later."
-msgstr "Un error ocurrió al cargar el video. Por favor intente denuevo."
+msgstr "Se produjo un error al cargar el video. Por favor, inténtelo de nuevo."
```

Comment:
IMHO 😄  `inténtelo` -> `intente`

Reply:
I vote for `inténtalo` 😄. Anyway, it should be `inténtalo` or `intenta` (_de tú, no usted_).